cfme/fixtures/pytest_selenium.py

```diff
@@ -600,6 +600,24 @@ def unset_attribute(loc, attr):
     return execute_script("arguments[0].removeAttribute(arguments[1]);", element(loc), attr)
 
 
+def set_angularjs_value(loc, value):
+    """Sets value of an element managed by angularjs
+
+    Args:
+        loc: A locator, expects either a string, WebElement, tuple.
+        value: Value to set.
+    """
+    logger.info("Setting value of an angularjs element {} to {}".format(loc, value))
+    setter_script = """\
+      var angular_elem = angular.element(arguments[0]);
+      var $parse = angular_elem.injector().get('$parse');
+      var getter = $parse(arguments[0].getAttribute('ng-model'));
+      var setter = getter.assign;
+      angular_elem.scope().$apply(function($scope) {{ setter($scope, '{}'); }});
+    """.format(value)
+    return execute_script(setter_script, element(loc))
```

Comment:
Right so I just confirmed that it indeed is undefined in the local scope so... I'd like to leave it as it is.